### PR TITLE
apps/projects: make sure predicate is checked against organisation al…

### PR DESCRIPTION
**Pull request by fuzzylogic2000**
…so when project given

fixes https://github.com/liqd/a4-meinberlin/issues/3419

### CODE REVIEW

File: adhocracy4/organisations/predicates.py [hunk 1]
```diff
@@ -26,9 +26,17 @@ def is_org_member(user, subject):
...
-def is_org_group_member(user, organisation):
-    if organisation:
-        if hasattr(organisation, 'groups'):
+def is_org_group_member(user, subject):
+    if subject:
+        if hasattr(subject, 'has_initiator') and hasattr(subject, 'groups'):
+            organisation = subject
+            org_groups = organisation.groups.all()
+            user_groups = user.groups.all()
+            group = org_groups & user_groups
```
Comment (Rineee) [inline adhocracy4/organisations/predicates.py:35]: Nitpick :) : What do you think about taking these out of the if statement and combining them with lines 40-42?

**Review by Rineee — APPROVED** [review]
Cool, that works perfectly! I made a small comment, but if you think its unnecessary, just merge!

File: adhocracy4/organisations/predicates.py (hunk 1 above)
Comment (fuzzylogic2000) [reply adhocracy4/organisations/predicates.py:35]: Haha, late brain only thought about the return outside of the ifs. But that can also go in the else! Yes, good point!

Comment (fuzzylogic2000) [reply adhocracy4/organisations/predicates.py]: And I didn't even need the else!

Comment (Rineee) [reply adhocracy4/organisations/predicates.py]: Perfect!

**Review by Rineee — APPROVED** [review]

**Pull request merged**